runtime/near-vm-runner/src/cache.rs

```diff
@@ -379,23 +377,3 @@ pub fn precompile_contract(
 ) -> Result<ContractPrecompilatonResult, ContractPrecompilatonError> {
     precompile_contract_vm(VMKind::default(), wasm_code, config, cache)
 }
-
```

Comment:
Please don't remove this code. It is intended way of doing things. We use this hash for persistent key generation, so cannot afford moving hash, and this solution hinted by @matklad is intended to ensure that hash never changes.